irispy/iris_tools.py

```diff
@@ -24,12 +24,16 @@
 # paper.
 DETECTOR_GAIN = {"NUV": 18., "FUV": 6., "SJI": 18.}
 DETECTOR_YIELD = {"NUV": 1., "FUV": 1.5, "SJI": 1.}
+BSCALE = 0.25
+BZERO = 7992
 DN_UNIT = {
     "NUV": u.def_unit("DN_IRIS_NUV", DETECTOR_GAIN["NUV"]/DETECTOR_YIELD["NUV"]*u.photon),
     "FUV": u.def_unit("DN_IRIS_FUV", DETECTOR_GAIN["FUV"]/DETECTOR_YIELD["FUV"]*u.photon),
-    "SJI": u.def_unit("DN_IRIS_SJI", DETECTOR_GAIN["SJI"]/DETECTOR_YIELD["SJI"]*u.photon)}
+    "SJI": u.def_unit("DN_IRIS_SJI", DETECTOR_GAIN["SJI"]/DETECTOR_YIELD["SJI"]*u.photon),
+    "SJI_UNSCALED": u.def_unit("DN_IRIS_SJI_UNSCALED",
+                               (DETECTOR_GAIN["SJI"]/DETECTOR_YIELD["SJI"]*BSCALE + BZERO)*u.photon)}
```

Comment:
Do you mean that I have to create a new function that convert SJI unit to SJI_UNSCALED unit and vice versa by using equivalences ?

Reply:
I created the below tuple :
`SJI_SCALING = (DN_UNIT["SJI"], DN_UNIT["SJI_UNSCALED"], lambda x: 7992.0*x,` 
`lambda x: -31964.0*x)`

Then, when I do 
`iris_tools.DN_UNIT['SJI'].to(iris_tools.DN_UNIT['SJI_UNSCALED'], equivalencies=iris_tools.SJI_SCALING)`

I have a false value returning (same value for different calculations).
`0.0022509848058525606`

The values 7992.0 and -31964.0 are correct but I will replaced them by default values when I will solved my bug, they help me to work faster. 

Do you have an idea about it ?

Reply:
Firstly, I think ```SJI_SCALING``` has to be a list of tuples.  In this case it would be a list of one tuple.  Not sure if that makes a difference.

Next, isn't the function to go from unscaled to scaled ```x * SJI_DEFAULT_BSCALE + SJI_DEFAULT_BZERO```?  And then the reverse would be ```(x - SJI_DEFAULT_BZERO) / SJI_DEFAULT_BSCALE```?  Am I wrong on this?

Reply:
Yes, I have these function but its not working ...